### DISCUSSION TITLE
DNS suddenly from nowhere stop working.

### DISCUSSION THREAD

**Discussion by maleuh1234** [vercel/community#6176] [Help] [answered]
### Summary

Today at 1pm, my mail and website suddenly stopped working. I made some research and from nowhere, my dns config is not ok anymore. I have the default config and I purchased my domain at vercel. 
My domain as been paid without any problem in January. 

Is it going to be back by itself or should I do something?

### Steps to Reproduce

It came from nowhere

**Comment by amyegan** [comment]
Hey @maleuh1234. What is the domain?

**Reply by maleuh1234** [reply]
maeldugue.ch

**Reply by rsalzer** [reply]
I have the same issue with: biastest.ch ... It stopped working yesterday. 
I purchased the domain through vercel and never changed the default DNS-settings.

**Reply by maleuh1234** [reply]
I got an answer, vercel does not give you the domains so they can't verify it so the company who is in charge supressed all the .ch from vercel... 

I bought another domain at infomaniak (ok with the swiss law so they don't delet it) and i'm trying to make it redirect to my project now, but I can't get the ip because vercel does not provide one ip for your website and the ip they give you change constantly :(

**Reply by rsalzer** [reply]
Wait ... how can Vercel sell domains that don't work for their services? That does not make any sense.

**Reply by maleuh1234** [reply]
the domains works perfectly but the swiss law says that the domain must be a propriety of an individual. switch, the company who is in charge of the .ch send an email to check if it was an individual and to send a proof for that and they didn't send any. So every single .ch at vercel as been deleted yesterday and will be available to purchase the 21'th April at 1am. Sorry to have to tell you that.

**Reply by maleuh1234** [reply]
the company told me "don't buy anything at vercel, they're scammers", because it's not an approved by the company and considerate as scammers.

**Reply by rsalzer** [reply]
What is your source for that information?
When I purchased the domain, I had to fill in my private address and this information should be visible in the records.

**Reply by maleuh1234** [reply]
I called the company, they told me that.

**Reply by rsalzer** [reply]
Did you buy the domain through vercel? Or which company do you mean?
Switch should have the information about sending out emails and deleting domains somewhere, no?

**Reply by rsalzer** [reply]
Hmm ... found it at nic.ch (Domain-Lookup):
![image](https://github.com/vercel/community/assets/17248257/97eabfe9-e071-4e80-be7c-ac6e17517e86)

I will write them.

**Reply by maleuh1234** [reply]
The one who got deleted has been buy on vercel, the new one on INFOMANIAK, and you have an option to redirect the domain to the ip of vercel.
I you want to know how, get the ip on "dig google" select A and past the url to your project (the long domain name you get by default) 
Then in infomaniak call the support and ask them to redirect to the 2 ip you will get and they will explain you how to do it.

Finnaly, add the domain name on vercel without having it on and it will work :)

**Reply by maleuh1234** [reply]
you can call them also, they are very kind.

**Comment by rsalzer** [comment]
Thats a workaround for me and not a solution ... if I buy something through Vercel I dont expect to have to deal with a 3rd party.
Especially when I pay for a service they should provide support for it. So I will try to contact them.

**Reply by maleuh1234** [reply]
Yes but the fact is you have lost your domain and have to buy it back and if you do it again on vercel, it's going to happen again

**Reply by rsalzer** [reply]
I reached out to vercel. They are investigating the issue.

> Hi Robert,
> 
> Thank you for reaching out to Vercel Support!
> 
> I'm really sorry for this experience regarding your domain is not working as expected.
> 
> We're currently investigating issues with some .ch domains and are working with the Swiss registry (SWITCH) on bringing affected domains back online with highest priority.
> 
> Once new information become available, we'll make sure to let you know.
> 
> Thank you for your patience.
> 
> 
> Best,
> 
> Sebastian
> ▲ Staff Customer Success Engineer at [Vercel](https://vercel.com/)

**Reply by stefanstranger** [reply]
Is this also impacting the vercel domains? My web app https://[azure-subnet-copilot.vercel.app](https://azure-subnet-copilot.vercel.app/)/ is also not available.

**Reply by cameronweibel** [reply]
@stefanstranger your domain works for me:

![image](https://github.com/vercel/community/assets/8155129/b5e8bed7-53d7-402d-be15-35acf5591f1a)

**Reply by sebastianroming** [reply]
> Is this also impacting the vercel domains? My web app https://[azure-subnet-copilot.vercel.app](https://azure-subnet-copilot.vercel.app/)/ is also not available.  

No, this only affected a few custom domains with the Swiss registry, none of any `.vercel.app` domains were affected.

**Reply by stefanstranger** [reply]
Hi Sebastian,

Just tested it and it's working again. No idea what the glitch was. 🙏

**Comment by cameronweibel** [comment]
The .ch domains should be working from 15:20 on today. I just called SWITCH.

**Reply by rsalzer** [reply]
Thanks! They seem to work again now! (CC: @maleuh1234)

**Comment by sebastianroming** [comment, marked as answer]
Domains are back and working as expected again.
